receiver/splunkhecreceiver/splunkhec_to_metricdata.go

```diff
@@ -143,7 +143,7 @@ func buildLabels(dimensions map[string]interface{}) pdata.StringMap {
 			// TODO: Log or metric for this odd ball?
 			continue
 		}
-		labels.Insert(key, fmt.Sprintf("%v", val))
+		attributes.InsertString(key, fmt.Sprintf("%v", val))
```

Comment:
Should this check the type of `val` and insert the appropriate typed value? If it's too much for this PR can you file an issue and link it here?

Reply:
@codeboten please file an issue.